cmake/HPX_GeneratePackage.cmake

```diff
@@ -53,6 +53,12 @@ foreach(dir ${HPX_LIBRARY_DIR})
   set(HPX_PKG_LIBRARY_DIR "${HPX_PKG_LIBRARY_DIR} -L${dir}")
 endforeach()
 
+find_program(FULL_PYTHON_CMD python3)
```

Comment:
We already have `PYTHON_EXECUTABLE` set to the python interpreter found by the build system.

Reply:
Also, we shouldn't explicitly limit things to Python V3.